localstripe/resources.py

```diff
@@ -2878,3 +2887,41 @@ def __init__(self, card=None, customer=None, **kwargs):
 
         self.type = 'card'
         self.card = card_obj
+
+
+class Balance(StripeObject):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        # All exceptions must be raised before this point.
+        super().__init__()
+
+        currencies = ['usd', 'eur']
+
+        self.type = 'balance'
+
+        # Set the data in the Stripe specification
+        # https://stripe.com/docs/api/balance/balance_object
+        # by default there are neither avaliable, nor pending funds
+        self.available = [
+            self._get_fund(amount=0, currency=currency)
+            for currency in currencies
+        ]
+        self.pending = [
+            self._get_fund(amount=0, currency=currency)
+            for currency in currencies
+        ]
+
+        schedule_webhook(Event('balance.available', self))
+
+    def _get_fund(self, amount, currency):
+        return {
+            'amount': amount,
+            'currency': currency,
+            'source_types': {
+                'card': amount
+            }
+        }
```

Comment:
I think it would be more readable to drop this function and just use:
```python
        self.pending = [{'amount': 0,
                         'currency': currency,
                         'source_types': {'card': 0}}
                        for currency in currencies]
```